virtual_rainforest/models/animals/animal_communities.py

```diff
@@ -41,6 +44,7 @@ def __init__(
         self.plant_community: PlantCommunity = PlantCommunity(10000.0, 1)
         self.carcass_pool: CarcassPool = CarcassPool(10000.0, 1)
         self.soil_pool: PalatableSoil = PalatableSoil(10000.0, 1)
+        self.excrement_pool: ExcrementPool = ExcrementPool(0.0)
```

Comment:
I think we're best off initializing at a non-zero value. If we've got small bodied excrement specialists and a call order that puts foraging before excretion, we could have complete extinction from starvation in the first monthly time-step.